groups/ntc/ntci/ntci_datagramsocketsession.h

```diff
@@ -83,6 +83,18 @@ class DatagramSocketSession
         const bsl::shared_ptr<ntci::DatagramSocket>& datagramSocket,
         const ntca::ReadQueueEvent&                  event);
 
+    /// Process the condition that the read queue rate limit has been reached
```

Comment:
s/into the read queue/from the receive buffer/. The rate limiting applies to our internal interaction with the operating system receive buffer, not with the user's interaction with our virtual read queue. Additionally, place a period at the end. Apply this suggestion wherever else is appropriate.

Reply:
Done